### PR TITLE
Feature/extractor info update date

### LINKED ISSUE #89: `POST /api/extractors` should ignore `update` field
The field should we be set internally so that a user cannot overwrite the update date of an extractor info document with an arbitrary date time.

Also when printing out and ExtractorInfo document the date fields should be printed using a standard format.

### PR DESCRIPTION
**Pull request by lmarini**
## Description
When a client currently does a `POST /api/extractors` it can provide a value for the `update` field. This shouldn't be allowed as this is an internal date to keep track of when the document was last updated.

This fix makes it so that if `update` is set, it will be ignored and it will be automatically be set to the current date time.
This also prints the update date when doing a `GET /api/extractors` using ISO08601.

Fixes #89. 

## Review Time Estimate

- [ ] Immediately
- [x] Within one week
- [ ] When possible

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [x] I have signed the CLA
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Script to clean extractors' tmp files.
 - Script for RabbitMQ error queue cleanup.
 - Ability to use basic html formatting in the welcome message on the home page. [#51](https://github.com/clowder-framework/clowder/issues/51)
+- Ignore the `update` field when posting to `/api/extractors`. [#89](https://github.com/clowder-framework/clowder/issues/89)
```
Comment (robkooper) [inline CHANGELOG.md:14]: looks like the changelog is old

File: app/models/Extraction.scala [hunk 2]
```diff
@@ -121,16 +124,21 @@ object ExtractorInfo {
     def reads(json: JsValue): JsResult[URL] = JsSuccess(new URL(json.toString()))
     def writes(url: URL): JsValue = Json.toJson(url.toExternalForm)
   }
+
   implicit val dateFormat = new Format[Date] {
+    val simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
```
Comment (robkooper) [inline app/models/Extraction.scala:129]: doesn't look like we use simplDateFormat

**Review by robkooper — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lmarini) [reply CHANGELOG.md:14]: it's going into `extractor-catalog-wireframe`. I just merged that branch in this one.

File: app/models/Extraction.scala (hunk 2 above)
Comment (lmarini) [reply app/models/Extraction.scala:129]: Thanks. I switched to using the function in `util.Parsers` at the last minute. Will fix.

**Review by bodom0015 — APPROVED** [review]
LGTM :+1: Works as advertised, and resolves the issue uncovered in #69

File: app/models/Extraction.scala (hunk 2 above)
Comment (lmarini) [reply app/models/Extraction.scala]: Removed.

**Pull request merged**